packages/flutter/lib/src/material/chip_theme.dart

```diff
@@ -351,10 +352,21 @@ class ChipThemeData with Diagnosticable {
   /// Defaults to 4 logical pixels on all sides.
   final EdgeInsetsGeometry padding;
 
+  /// The color and weight of the chip's outline.
+  ///
+  /// If null, defaults to the value set by [shape].
```

Comment:
Not clear what "set by [shape]" means. I assume you mean the shape's side?

Reply:
Updated to: `If null, defaults to the border side of [shape].`